certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -436,25 +436,13 @@
           "200": {
             "description": "default",
             "schema": {
-              "type": "object",
-              "properties": {
-                "pageNumber": {
-                  "type": "integer",
-                  "format": "int32",
-                  "description": "Page Number",
-                  "title": "Page Number"
-                },
-                "dataList": {
-                  "type": "array",
-                  "items": {
-                    "$ref": "#/definitions/ExamScoreResultData"
-                  },
-                  "description": "dataList"
-                }
+              "type":"array",
```

Comment:
@RjImpx The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please confirm that this will not affect any users?